package.json

```diff
@@ -11,7 +11,7 @@
     "prod": "ENV=prod node -r ./prod-paths.js ./dist/src/server.js",
     "typeorm": "ts-node -r tsconfig-paths/register ./node_modules/typeorm/cli.js",
     "posttypeorm": "prettier -w src/database/migrations",
-    "test": "ENV=test jest --runInBand --forceExit --detectOpenHandles --useStderr",
+    "test": "ENV=test jest --runInBand --forceExit --detectOpenHandles --useStderr --silent",
```

Comment:
Do we need to hide the successfully tests?

Reply:
Nope, Ill remove it haha, thanks. 👯‍♂️